definitions/divorce/json/CaseEvent/CaseEvent-resp-journey-roles-and-permissions-nonprod.json

```diff
@@ -8,6 +8,19 @@
     "DisplayOrder": 1,
     "PreConditionState(s)": "*",
     "PostConditionState": "*",
-    "SecurityClassification": "Public"
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y"
```

Comment:
Hi @jacmurphy,
You had forgotten that in your story. You need it to show the summary page :)

Reply:
@adamjhc